grakn-engine/src/main/java/ai/grakn/engine/loader/LoaderTask.java

```diff
@@ -59,15 +59,16 @@
     private final QueryBuilder builder = Graql.withoutGraph().infer(false);
 
     @Override
-    public void start(Consumer<String> saveCheckpoint, Json configuration) {
+    public boolean start(Consumer<String> saveCheckpoint, Json configuration) {
         attemptInsertions(
                 getKeyspace(configuration),
                 getInserts(configuration));
+        return true;
```

Comment:
Ahh that is my confusion then. So it's on the todo of making loader tasks stoppable ?